components/servo/lib.rs

```diff
@@ -130,6 +130,29 @@ mod media_platform {
     use super::ServoMedia;
     use servo_media_gstreamer::GStreamerBackend;
 
+    #[cfg(target_os = "windows")]
+    fn set_gstreamer_log_handler() {
+        use gstreamer::{debug_add_log_function, debug_remove_default_log_function, DebugLevel};
+
+        debug_remove_default_log_function();
+        debug_add_log_function(|_, level, file, function, line, _, message| {
+            let message = format!(
+                "{:?} {:?}:{:?}:{:?} {:?}",
+                level, file, line, function, message
```

Comment:
Isn't this missing the debug category? That's quite important for figuring out which part of gstreamer is sending the message. See: https://gitlab.freedesktop.org/nirbheek/gst-uwp-test-app/blob/master/hello-world/MainPage.xaml.cpp#L78